src/service/vsync-impl.js

```diff
@@ -416,6 +419,23 @@ export class Vsync {
 
 
 /**
+ * For optimization reasons to stop try/catch from blocking optimization.
+ * @param {function(!VsyncStateDef)|undefined} callback
```

Comment:
I don't think it can be `undefined`, we check specifically for `task.measure` (and `mutate`) before calling.